announcements/2020-Week-32/aws.md

```diff
@@ -0,0 +1,16 @@
+# :zap: This Week in AWS Releases (August 7, 2020):zap:
+
+[AWS v11.5.1](https://github.com/giantswarm/releases/tree/master/aws/v11.5.1) fixes a problem with the calculation of the maximum number of Pods per node and improves resource cleanup after cluster deletion.
+
+[AWS v12.1.0](https://github.com/giantswarm/releases/tree/master/aws/v12.1.0) upgrades Cert Manager. (Discussion with SE prior to ugprade is recommended.)
```

Comment:
Note: This is already deprecated and 12.1.1 is in the making. Will probably be ready before the announcement deadline.

Reply:
@marians did you announce to customers that this release is being deactivated? I think either we (1) eliminate this here or (2) use it as a way to announce that we deactivated. wdyt?

Reply:
Definitely remove it here, please, independent of any decision regarding announcing the deprecation.